src/ffi.h

```diff
@@ -13,3 +14,18 @@ inline std::shared_ptr<T> box_to_shared_ptr(rust::Box<T> &&value) {
     std::shared_ptr<T> shared(ptr, [](T *ptr) { rust::Box<T>::from_raw(ptr); });
     return shared;
 }
+
+inline static rust::Vec<wcharz_t> to_rust_string_vec(const wcstring_list_t &strings) {
+    rust::Vec<wcharz_t> rust_strings;
+    rust_strings.reserve(strings.size());
+    for (const wcstring &string : strings) {
+        rust_strings.emplace_back(string.c_str());
+    }
+    return rust_strings;
+}
+
+inline static void trace_if_enabled(const parser_t & parser, wcharz_t command, const wcstring_list_t& args) {
```

Comment:
nit: should be formatted with `clang-format`

Reply:
Thanks